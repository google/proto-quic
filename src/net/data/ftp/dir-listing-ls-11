total 14
drwxr-xr-x   2 other        512 Feb 25  2009 beid
lrwxrwxrwx   1 bin            9 May  1  2007 bin -> ./usr/bin
d--x--x--x   2 sys          512 May  1  2007 dev
d--x--x--x   5 sys          512 May  1  2007 etc
drwxr-xr-x   2 sys          512 Mar 27  2009 pub
drwxr-xr-x   3 other        512 Apr 11  2007 tigerd1
d--x--x--x   6 sys          512 May  1  2007 usr
