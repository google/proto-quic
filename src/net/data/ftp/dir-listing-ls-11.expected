d
beid
-1
2009
2
25
0
0

l
bin
-1
2007
5
1
0
0

d
dev
-1
2007
5
1
0
0

d
etc
-1
2007
5
1
0
0

d
pub
-1
2009
3
27
0
0

d
tigerd1
-1
2007
4
11
0
0

d
usr
-1
2007
5
1
0
0
